sorting/bubbleSort.js

```diff
@@ -0,0 +1,21 @@
+const bubbleSort = (arr) => {
+
+    if(arr !== null) {
+        let len = arr.length;
```

Comment:
this is good that you stored the length value but would have been great to have inside for loop declaration itself.

Reply:
length of array will remain same so stored the value outside loop so that it wont calculate every time in the loop.